modules/apps/transfer/types/packet.go

```diff
@@ -34,6 +34,20 @@ func NewFungibleTokenPacketData(
 	}
 }
 
+func NewFungibleTokenPacketDataWithMetadata(
```

Comment:
We should break the API of NewFungibleTokenPacketData. In backports we could include this function, but maybe it has no benefit if we remove it in v6. Thoughts?

Reply:
that sounds good to me. Not sure what the process for this is, do we create different PRs for each version?

Reply:
We will add the backport (for a new minor version on each supported major release line) tags to this pr. Once merged, mergify will try to automatically backport this to the release line. At this point we can manually modify each backport to remove any API breaking changes. This is something we can handle